### ISSUE TITLE
Website domain blocked for no apparent reason

### ISSUE THREAD

**Issue by Nemojon** [phantom-labs/blocklist#113]
Hi there, my domain solswipe.io has been blocked for no apparent reason. 
We have issued and submitted countless documents and even implemented 4 milestone escrow to our mint. So there is technically close to no way that we can perform any funny actions as all our funds will be locked and only unlocked if our community releases it. We have also submitted lots of documents and will continue posting up more information when we have it. And also postponed our mint to september specifically for the intention of providing more supporting documents to ME or community before our mint. 

Therefore,
I would like to appeal that the domain gets unblocked thank you.

**Comment by Harii94** [comment]
Hi, 
We have merged the pull req to remove your website from the blocklist.
